includes/commerce_pos.payment.inc

```diff
@@ -40,11 +48,12 @@ function commerce_pos_payment(&$form, &$form_state, $transaction_type) {
 
   form_load_include($form_state, 'inc', 'commerce_pos', 'includes/commerce_pos.common');
   commerce_pos_payment_ajax_check($form, $form_state);
+  dpm($form_state);
 
   $return = $transaction_type == CommercePosService::TRANSACTION_TYPE_RETURN;
   $payment_options = commerce_pos_get_payment_options();
   $form['#theme'] = 'commerce_pos_payment';
-  $form['#redirect_key'] = $return ? 'return' : 'sale';
+  $form['#redirect_key'] = $return ? 'return' : ($transaction_type == CommercePosService::TRANSACTION_TYPE_EXCHANGE ? 'exchange' : 'sale');
   $form['#transaction_type'] = $transaction_type;
 
   $form['#prefix'] = '<div id="' . $wrapper_id . '" class="' . ($return ? 'return' : 'sale') . '">';
```

Comment:
In this case, the class will be sale even for exchanges. Is this what you intended?